PREP_SYSTEM_FOR_DIETPI.sh

```diff
@@ -420,8 +413,7 @@
 
 		)
 
-		G_WHIP_MENU 'Please select an option:'
-		if (( ! $? && $G_WHIP_RETURNED_VALUE == 1 )); then
+		if G_WHIP_MENU 'Please select an option:' && G_WHIP_RETURNED_VALUE==1 ; then
```

Comment:
`G_WHIP_RETURNED_VALUE==1` was no assignment here, but a value check.
Needs to be `(( $G_WHIP_RETURNED_VALUE == 1 ))` or simply `(( $G_WHIP_RETURNED_VALUE ))`.

Btw `$` inside arithmetic is not needed to call variables, but we do this as sort of consistency ( to `[[ $VAR ]]` ) and to allow searching for all variable calls within editor via `$VAR` pattern 😉.